### PR TITLE
fix for #2 and #3

### PR DESCRIPTION
**Pull request by lambdaburrito**

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -62,14 +62,29 @@ fn get_num_cpus() -> usize {
...
 #[test]
-fn it_works() {
+fn lower_bound() {
     assert!(get() > 0);
 }
+
+
+#[test]
+fn upper_bound() {
+    assert!(get() < 236,451);
```
Comment (seanmonstar) [inline src/lib.rs]: You can't actually use commas in number literals like this. You *can* however use underscores. I'll just merge and fix myself to `236_451`.

Comment (seanmonstar) [inline src/lib.rs]: That this compiles is because it's actually reading it as `get() < 236`, and if assertion is wrong, display message is `451`.

Comment (sinistersnare) [inline src/lib.rs]: Thats hilarious, good thing its an easy fix :D

Comment (lambdaburrito) [inline src/lib.rs]: LOL I would like to claim that it was intentional trickery but it was just me copy & pasting from my Mathematica shell.

**Pull request merged**